CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# Unreleased
+
+Please use this place to add information of work that hasn't been released,
+and specify if that is backwards-compatible.
+
+# 30.6.0
+
+* Use `document_type` & `schema_name` in special routes
+* Remove implicit dependency between JsonClient and GdsApi::Base
```

Comment:
This should include a note for https://github.com/alphagov/gds-api-adapters/pull/505 :)